chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+The women of Dressara often wore dresses, which Leanne both admired and found a shame. Leanne enjoyed wearing dresses, but found women in suits much more handsome.
+
+Delphine was nothing if not handsome. She was tall, and did her best to look so: her fine Elisian suit lengthened her torso, and she had paired it, rather than with pants, with a skirt just long enough to wave gently in the light breeze the Dressarans were so fond of.
+
+Dressara was a large moon. It was somewhat pleasant, if you enjoyed their sort of pomp, unlike the dour planet Seven around which it held a lazy orbit. Like anywhere else, it was rather difficult to land on illegally, particularly when it was in the process of adjusting its orbit. 
```

Comment:
Dressara was a large moon. It was somewhat pleasant, if you enjoyed their sort of pomp, unlike the dour planet Seven around which it held its lazy orbit. Like anywhere else, it was nearly impossible to land on illegally. 

The traffic around Dressara lately was chaotic: the planet’s orbit was currently being altered, which wasn’t an issue in its own right; however, all of its many satellites required adjustments likewise, and there had apparently been many issues. Many of the satellites had required replacement energy cells to power their shift, but the energy cell shipment allocated for them had gone mysteriously gone missing.

Even under normal conditions, however, Dressara still had the same double-layered defense system as anywhere else.

It was simpler to land legally. While Leanne didn’t have an Alt to identify her, nor to navigate her ship around the mess of traffic around Dressara, she did have a collection of bounty cards for people on all the various planets and moons. Their targets were now long dead, but so long as the bodies weren’t found Leanne could use their bounties to request both permission to land and an opening in the mess through which to do so.